extensions/amp-analytics/analytics-vars.md

```diff
@@ -206,6 +206,46 @@ is unloaded. The value is in milliseconds.
 
 Example value: `40`
 
+### navRedirectCount
+
+Provides the number of redirects since the last non-redirect navigation.
+See W3C Navigation Timing API [PerformanceNavigation interface](https://www.w3.org/TR/navigation-timing/#performancenavigation) for more information.
+
+Example value: `0`
+
+### navTiming
+
+Provides access to metrics from the browser's PerformanceTiming interface.
+If both `start-event` and `end-event` arguments are passed, the value will be the time elapsed between the two events.
+Otherwise, if only `start-event` argument is passed, the value will be the timestamp of the given event.
+The value is in milliseconds.
+
+See the W3C Navigation Timing API [PerformanceTiming interface](https://www.w3.org/TR/navigation-timing/#sec-navigation-timing-interface) documentation for attribute names and definitions.
+
+Please see below for the required and optional arguments you may pass into `navTiming` like a function. Spaces between arguments and values are not allowed.
+
+**arguments**:
+
+  - `start-event` (Required) - Name of the PerformanceTiming interface attribute corresponding to the start event.
+
+  - `end-event` (Optional) - Optional name of the PerformanceTiming interface attribute corresponding to the end event. If `end-event` is passed, the value will be the time difference between the start and end events.
+
+
+Example 1 usage: `${navTiming(navigationStart)}`
```

Comment:
nit: Maybe reword this as : 

Example 1: `${navTiming(navigationStart)}` results in value: `1451606400000`

Example 2:...